### ISSUE TITLE
Aws::S3::Errors::AllAccessDisabled: All access to this object has been disabled

### ISSUE THREAD

**Issue by attahkhan016** [alienfast/elastic-beanstalk#47]
i am getting this error : 
i have given the correct access keys and my eb_config  looks like this

```
app: acme
region: us-east-1
solution_stack_name: 64bit Amazon Linux 2015.03 v2.0.0 running Ruby 2.2 (Passenger Standalone)
strategy: inplace-update 
keep_latest: 10
development:
  options:
    aws:autoscaling:launchconfiguration:
      InstanceType: t1.micro

production:
  strategy: blue-green 
  options:
    aws:autoscaling:launchconfiguration:
      InstanceType: t1.small
```

but when i am trying to deploy it , i am getting this error : 

```
[2016-07-01 06:12:55 UTC][package:acme.zip] start uploading to s3 bucket acme.packages...
rake aborted!
Aws::S3::Errors::AllAccessDisabled: All access to this object has been disabled
/home/bitlasoft/.rvm/gems/ruby-2.3.0/gems/aws-sdk-core-2.3.18/lib/seahorse/client/plugins/raise_response_errors.rb:15:in `call'
/home/bitlasoft/.rvm/gems/ruby-2.3.0/gems/aws-sdk-core-2.3.18/lib/aws-sdk-core/plugins/s3_sse_cpk.rb:19:in `call'
/home/bitlasoft/.rvm/gems/ruby-2.3.0/gems/aws-sdk-core-2.3.18/lib/aws-sdk-core/plugins/s3_accelerate.rb:33:in `call'
/home/bitlasoft/.rvm/gems/ruby-2.3.0/gems/aws-sdk-core-2.3.18/lib/aws-sdk-core/plugins/param_converter.rb:20:in `call'
/home/bitlasoft/.rvm/gems/ruby-2.3.0/gems/aws-sdk-core-2.3.18/lib/seahorse/client/plugins/response_target.rb:21:in `call'
/home/bitlasoft/.rvm/gems/ruby-2.3.0/gems/aws-sdk-core-2.3.18/lib/seahorse/client/request.rb:70:in `send_request'
/home/bitlasoft/.rvm/gems/ruby-2.3.0/gems/aws-sdk-core-2.3.18/lib/seahorse/client/base.rb:207:in `block (2 levels) in define_operation_methods'
/home/bitlasoft/.rvm/gems/ruby-2.3.0/gems/aws-sdk-resources-2.3.18/lib/aws-sdk-resources/services/s3/file_uploader.rb:42:in `block in put_object'
/home/bitlasoft/.rvm/gems/ruby-2.3.0/gems/aws-sdk-resources-2.3.18/lib/aws-sdk-resources/services/s3/file_uploader.rb:49:in `open_file'
/home/bitlasoft/.rvm/gems/ruby-2.3.0/gems/aws-sdk-resources-2.3.18/lib/aws-sdk-resources/services/s3/file_uploader.rb:41:in `put_object'
/home/bitlasoft/.rvm/gems/ruby-2.3.0/gems/aws-sdk-resources-2.3.18/lib/aws-sdk-resources/services/s3/file_uploader.rb:34:in `upload'
/home/bitlasoft/.rvm/gems/ruby-2.3.0/gems/aws-sdk-resources-2.3.18/lib/aws-sdk-resources/services/s3/object.rb:251:in `upload_file'
/home/bitlasoft/.rvm/gems/ruby-2.3.0/gems/eb_deployer-0.6.6/lib/eb_deployer/aws_driver/s3_driver.rb:18:in `upload_file'
/home/bitlasoft/.rvm/gems/ruby-2.3.0/gems/eb_deployer-0.6.6/lib/eb_deployer/package.rb:30:in `upload_if_not_exists'
/home/bitlasoft/.rvm/gems/ruby-2.3.0/gems/eb_deployer-0.6.6/lib/eb_deployer/package.rb:10:in `upload'
/home/bitlasoft/.rvm/gems/ruby-2.3.0/gems/eb_deployer-0.6.6/lib/eb_deployer/application.rb:22:in `create_version'
/home/bitlasoft/.rvm/gems/ruby-2.3.0/gems/eb_deployer-0.6.6/lib/eb_deployer.rb:232:in `deploy'
/home/bitlasoft/.rvm/gems/ruby-2.3.0/gems/elastic-beanstalk-1.2.1/lib/elastic/beanstalk/tasks/eb.rake:348:in `block (2 levels) in <top (required)>'
/home/bitlasoft/.rvm/gems/ruby-2.3.0/gems/rake-11.2.2/exe/rake:27:in `<top (required)>'
/home/bitlasoft/.rvm/gems/ruby-2.3.0/bin/ruby_executable_hooks:15:in `eval'
/home/bitlasoft/.rvm/gems/ruby-2.3.0/bin/ruby_executable_hooks:15:in `<main>'
Tasks: TOP => eb:deploy
(See full trace by running task with --trace)
```

where have i gone wrong ??

**Comment by rosskevin** [comment]
It looks like the message is telling you that it cannot create or modify `acme.packages` in your account:

```
[2016-07-01 06:12:55 UTC][package:acme.zip] start uploading to s3 bucket acme.packages...
rake aborted!
Aws::S3::Errors::AllAccessDisabled: All access to this object has been disabled
```

**Comment by attahkhan016** [comment]
but how can i rectify this /???


On Fri, Jul 1, 2016 at 7:08 PM, Kevin Ross <notifications@github.com> wrote:

> Closed #47 <https://github.com/alienfast/elastic-beanstalk/issues/47>.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/alienfast/elastic-beanstalk/issues/47#event-710502536>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ATFd912ldPVzRBPt5DiIGIAsVSX4x9OJks5qRRhngaJpZM4JC2aF>
> .
>



-- 
Thanks & Regards,

Name: Mohammed Attaullah Khan
email id: attahkhan016@gmail.com
contact number:+91-8123030179

**Comment by rosskevin** [comment]
You need to learn about AWS and setup your permissions. That is outside the scope of this gem or issue tracker. You may try asking on stack overflow.
